chromium_src/chrome/common/resource_usage_reporter.mojom

```diff
@@ -0,0 +1,39 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
```

Comment:
I wanted to keep it for posterity and in case we get the ability to build mojo bindings directly in the future (which we'll probably need because Chromium itself is moving towards making everything based on mojo), but I can :fire: it for now